client/state/jetpack-connect/actions.js

```diff
@@ -195,11 +196,44 @@ export function retryAuth( url, attemptNumber ) {
 	};
 }
 
+export function createSocialAccount( socialInfo ) {
+	return dispatch => {
+		dispatch( recordTracksEvent( 'calypso_jpc_social_create_account' ) );
```

Comment:
Not `calypso_jpc_create_social_account`?

Reply:
Should probably be `calypso_jpc_social_createaccount` to match other events.